src/main/java/com/google/sps/CloudVisionAPI.java

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+
+/**
+ * Interface used for Cloud Vision API related functionality.
+ */
+public interface CloudVisionAPI {
+  protected BatchAnnotateImagesResponse querier(List<AnnotateImageRequest> requests)
```

Comment:
I think the method should be public because you may want this method to be used by classes outside of your package.

https://stackoverflow.com/questions/3058976/java-protected-method-accessibility/3059239

Reply:
My ImageTextDetector is in the same package as CloudVisionAPI which is the outermost folder. and also Raluca's files which use these are in the `sps` package, so I thought it was okay to make it protected.